libqwebthree/CMakeLists.txt

```diff
@@ -37,5 +37,5 @@ target_link_libraries(${EXECUTABLE} ${JSON_RPC_CPP_SERVER_LIBRARIES})
 target_link_libraries(${EXECUTABLE} ethereum)
 target_link_libraries(${EXECUTABLE} secp256k1)
 
-install( TARGETS ${EXECUTABLE} ARCHIVE DESTINATION lib LIBRARY DESTINATION lib )
+install( TARGETS ${EXECUTABLE} RUNTIME DESTINATION bin ARCHIVE DESTINATION lib LIBRARY DESTINATION lib )
```

Comment:
DLLs were not getting copied to /bin. MSYS typically has static .a and dll.a link libraries in /lib, headers in /include, and executables and DLLs in /bin. Currently none of the DLLS are getting copied upon install.